types/api/xverse/ordinals.ts

```diff
@@ -38,25 +38,82 @@ export interface InscriptionInCollectionsList {
   data: Array<Inscription>;
 }
 
-export type SatRarity = 'common' | 'uncommon' | 'rare' | 'epic' | 'legendary' | 'mythic';
+export const RoadArmorRareSats = ['MYTHIC', 'LEGENDARY', 'EPIC', 'RARE', 'UNCOMMON', 'COMMON'] as const;
+export type RoadArmorRareSatsType = (typeof RoadArmorRareSats)[number];
 
-export type BundleRareSat = {
-  number: string;
-  rarity_ranking: SatRarity;
-  offset: number;
-};
+export const Sattributes = [
```

Comment:
nit: we use "satribute" in api code

Reply:
Yeah, I initially wrote it sattributes everywhere as well because English 😋 All the market places use a single 't' though, so we went with that for consistency.